Audio Mixer Demo/ViewController.swift

```diff
@@ -30,25 +30,28 @@ class ViewController: UIViewController {
         // this is just a demo so I'll let you off.
         // Here you are getting the path for the sound file you added to your project and converting it into a file url.
         let path = NSBundle.mainBundle().pathForResource("farah-faucet", ofType: "wav")!
-        let url = NSURL.fileURLWithPath(path)!
+        let url = NSURL.fileURLWithPath(path)
         
         // Here you are creating an AVAudioFile from the sound file,
         // preparing a buffer of the correct format and length and loading
         // the file into the buffer.
-        var file = AVAudioFile(forReading: url, error: nil)
-        var buffer = AVAudioPCMBuffer(PCMFormat: file.processingFormat, frameCapacity: AVAudioFrameCount(file.length))
-        file.readIntoBuffer(buffer, error: nil)
+        let file = try? AVAudioFile(forReading: url)
+        let buffer = AVAudioPCMBuffer(PCMFormat: file!.processingFormat, frameCapacity: AVAudioFrameCount(file!.length))
+        do {
+            try file!.readIntoBuffer(buffer)
+        } catch _ {
+        }
```

Comment:
I think just using `try!` would be more in keeping with the code in the rest of the demo, even though, as noted, using forced unwrapping is bad form it is OK for clarity in demo code (and catching but ignoring the error isn't really any better)